books/scss/styles.scss

```diff
@@ -41,3 +41,8 @@ body {
   font-size: 20px;
   line-height: 24px;
 }
+
+.link-1 {
```

Comment:
Adding numbers to the class names is not a really good practice. Try to avoid that please 😄

Reply:
Have been improved